go.mod

```diff
@@ -187,7 +187,7 @@ require (
 replace (
 	// patch bech32 decoding to enable larger string lengths
 	github.com/btcsuite/btcutil => github.com/kava-labs/btcutil v0.0.0-20200522184203-886d33430f06
-	github.com/cosmos/cosmos-sdk => github.com/ununifi/cosmos-sdk v0.47.1
+	github.com/cosmos/cosmos-sdk => github.com/ununifi/cosmos-sdk v0.47.1-bank-rc2
```

Comment:
Why does it require custom bank? I see following commit on modified sdk
https://github.com/UnUniFi/cosmos-sdk/commit/62acccf557cc0577560999442fe7232a18f0707b#diff-9d2eab3427e6248af17c3913110878659d8128d050b8476e609ae7c0e95b702dR30

I think this configuration can be modified on app.go without using custom cosmos-sdk.

Reply:
this is @KimuraYu45z  request.
I modifyed bank send func.
so user can't bank send except ununifi team

Reply:
I see - okay fine.